### ISSUE TITLE
Cart is emptied on checkout rather than on payment success

### ISSUE THREAD

**Issue by Buzut** [PaylineByMonext/payline-woocommerce#12]
This behaviour is undesired as customers have an empty cart when payment fails.

**Comment by JoKolov** [comment]
Any news about that, it's the same if payment is cancelled by customer ?

**Comment by Buzut** [comment]
Nope. As you can see, they haven't updated the plugin nor accept PR. 
I've been told it wasn't developed in-house and therefore I guess they don't actively maintain it as long as it more or less works.
